python-threatexchange/threatexchange/exchanges/clients/ncmec/hash_api.py

```diff
@@ -131,6 +144,7 @@ class NCMECEntryUpdate:
     deleted: bool
     classification: t.Optional[str]
     fingerprints: t.Dict[str, str]
+    feedback: t.List[t.Dict[str, t.Any]]
```

Comment:
(note to self): Consider whether it makes sense to use the enum here, or whether to leave as string in all cases